ethereum/contracts/pyth/PythInternalStructs.sol

```diff
@@ -43,11 +43,18 @@ contract PythInternalStructs {
         uint64 prevConf;
     }
 
-    struct PriceInfo {
-        uint256 attestationTime;
-        uint256 arrivalTime;
-        uint256 arrivalBlock;
-        PythStructs.PriceFeed priceFeed;
+ struct PriceInfo {
+        // slot 1
+        int64 price;
+        uint64 conf;
+        uint64 publishTime;
+        int32 expo;
+
```

Comment:
Could we abstract out the slot 1 and slot 2 into seperate structs? They both essentially represent a price+conf+timestamp. So we would have 
```
struct Price {
    int64 price;
    uint64 conf;
    uint64 publishTime;
    int32 expo;
}
struct PriceInfo {
    price Price;
    emaPrice Price;
}
```
`Price` isn't the greatest name but you get the idea.

Reply:
I don't know why but after making such a change the gas usage was increased. Its not much 0.37-0.68% change but let's keep it the way it is as I think we can bear it.

```
testBenchmarkUpdatePriceFeedsFresh() (gas: 1558 (0.377%)) 
testBenchmarkUpdatePriceFeedsNotFresh() (gas: 1462 (0.379%)) 
testBenchmarkUpdatePriceFeedsIfNecessaryFresh() (gas: 1846 (0.423%)) 
testBenchmarkGetPrice() (gas: 142 (0.527%)) 
testBenchmarkUpdatePriceFeedsIfNecessaryNotFresh() (gas: 1374 (0.684%)) 
```

Reply:
Ok, let me actually keep the clean version. Then I'll revert this change later if needed.